dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml

```diff
@@ -1,7 +1,8 @@
 %p
-  Welcome to Code.org! You have successfully linked your email to your child’s account.
-  We're thrilled that you have taken the next step in supporting #{@student.name} in their learning and exploration of
-  computer science. If you are not #{@student.name}’s parent, please reply to this email or contact support@code.org.
+  Welcome to Code.org! You have successfully linked your email to the account of your child,
+  %strong>= " #{@student.name}"
```

Comment:
is there supposed to be `>=` ?

Reply:
Yes. It felt weird to me to bold the period but haml adds a space between lines. The `>` disables the automatic whitespace for this line (which is also why there's a space before the student name). I'm open to suggestions on the best way to do this!